plugins/web/external/Testing_for_Command_Injection@OWTF-DV-013.py

```diff
@@ -37,6 +37,6 @@
 
 def run(Core, PluginInfo):
 	#Core.Config.Show()
-	Content = Core.PluginHelper.HtmlString("Intended to show helpful info in the future")
+	Content = Core.PluginHelper.ResourceLinkList('Online Resources', Core.DB.Resource.GetResources('CommandInjection'))
```

Comment:
The resource link list is good for the link but for a command, you need to use a [command box](https://github.com/owtf/owtf/blob/develop/framework/plugin/plugin_helper.py#L71)